charts/newrelic-infrastructure/Chart.yaml

```diff
@@ -8,7 +8,7 @@ sources:
   - https://github.com/newrelic/nri-kubernetes/tree/main/charts/newrelic-infrastructure
   - https://github.com/newrelic/infrastructure-agent/
 
-version: 3.12.0
+version: 3.12.1
 appVersion: 3.6.0
 
```

Comment:
If you also want to get the changes from the other 2 prs, https://github.com/newrelic/nri-kubernetes/pull/625 and https://github.com/newrelic/nri-kubernetes/pull/624/files you need to merge them first, create a new release (3.7.0 in this case as I understand it would be a Minor, because the dependencies are being bumped to a minor) and then change here this appVersion to 3.7.0 so the new image is used.
But I think you commented that you will wait until the new metric additions are merged, Am I right?

Reply:
Yes, I was planning on holding off on a release until I finish adding the new KSM metrics.